doc/rst/conf.py

```diff
@@ -75,13 +75,13 @@ def setup(app):
 # 'version' adds a redundant version number onto the top of the sidebar
 # automatically (rtd-theme). We also don't use |version| anywhere in rst
 
-chplversion = '1.26'                  # TODO -- parse from `chpl --version`
+chplversion = '1.25.1'                  # TODO -- parse from `chpl --version`
```

Comment:
I think that this should probably just be '1.25'.  Specifically, we tend not to support multiple versions of the docs for a given release (to save web space, to improve Google's ability to find our "latest" documents in its indexing), and I'm not sure what might blow up with an update version in this string.  Unless you can find a precedent for it in the file's history that I'm forgetting about, I think we should drop the '.1'.